packages/scene-composer/src/hooks/useBindingQueries.ts

```diff
@@ -0,0 +1,37 @@
+import { useMemo } from 'react';
+import { Query, TimeSeriesData } from '@iot-app-kit/core';
+import { isEmpty } from 'lodash';
+
+import { useSceneComposerId } from '../common/sceneComposerIdContext';
+import { IValueDataBinding } from '../interfaces';
+import { useStore } from '../store';
+
+/**
+ * Create query objects for data bindings.
```

Comment:
What does data binding refer to in this context? It's a general term yet seems to be referring to a specific data type utilized by the system. Would it be helpful to expand the comment here with additional info into what the data bindings are?

Reply:
Data binding is a specific feature in the context of the scene composer, or even in TwinMaker. It is a set of ids (currently support entityId, componentName, propertyName) to uniquely identify a data to be used by scene widgets to change their visual state. This hook is supposed to be used internally only, so whoever use it should know the definition.